optionals/cache_disable/config.cpp

```diff
@@ -7,7 +7,8 @@ class CfgPatches {
         requiredVersion = REQUIRED_VERSION;
         requiredAddons[] = {"CBA_Extended_EventHandlers", "CBA_Main"};
         version = VERSION;
-        author[] = {"Sickboy"};
+        author = "$STR_CBA_Author";
```

Comment:
Right, that is true. My bad, I am used to having it in one PBO and use `ECSTIRNG(pbo,Author)`.